nemo/collections/tts/g2p/models/zh_cn_pinyin.py

```diff
@@ -67,7 +75,9 @@ def __init__(
             apply_to_oov_word=apply_to_oov_word,
             mapping_file=mapping_file,
         )
-        self.tones = {'1': '#1', '2': '#2', '3': '#3', '4': '#4', '5': '#5'}
+
+        self.tones = {str(x): tone_prefix + str(x) for x in range(1, 6)}
```

Comment:
I guess we expect users to pass in an empty string if they don't want `tone_prefix`, but should we error-check in case they put in `None`?

Reply:
good point! will add the None check.